c2corg_ui/templates/i18n.html

```diff
@@ -118,6 +118,9 @@
 <span translate>You have no rights to edit this document.</span>
 <span translate>Form is not valid</span>
 <span translate>You must log in to edit this document.</span>
+<span translate>no associated routes</span>
```

Comment:
in this case I think it's ok - the message will appear as "Missing field: no associated route" and looks like a normal sentence. In contrast to the other messages with capital letters that are ugly, like "Missing field: Date start".